tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/compare_pgm_dispatch_perf_ci.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Run the pgm dispatch sweep
+cd $TT_METAL_HOME
+./tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/sweep_pgm_dispatch.sh | tee log
+./tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/filt_pgm_dispatch.pl log > results.log
+
+LOG_FILE1="$TT_METAL_HOME/tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/pgm_dispatch_golden.log"
+LOG_FILE2="results.log"
+THRESHOLD=5 # Percentage difference threshold
```

Comment:
@pgkeller Setting a 5% threshold for regressions, could be tighter.

Reply:
in my experience these tests are pretty deterministic.  Guessing we could go tighter and loosen if it becomes a problem.  Say 2.5%?